src/main/resources/assets/stylesheets/rwt.hbs

```diff
@@ -30,7 +30,7 @@ padding: 5px;
 .border-defects{
 border: 1px solid;
 padding: 1px 2px 2px 1px;
-height: 70px;
+min-height: 70px;
```

Comment:
will need max-height too if you want it to not push out your footer

Reply:
I want it to push the footer as it makes the content really ugly when the defects overflow. I tried that then chose min-height.